src/adapter/src/coord.rs

```diff
@@ -992,6 +1079,9 @@ impl<S: Append + 'static> Coordinator<S> {
             tokio::time::interval(self.catalog.config().timestamp_frequency);
         // Watcher that listens for and reports compute service status changes.
         let mut compute_events = self.controller.watch_compute_services();
+        // Interval for updating read holds on all objects
+        let mut update_read_holds =
```

Comment:
Is there an argument for why this shouldn't be in the advance timeline fn? I think it makes sense to tie that logic together: every time we move uppers forward, we also try to move sinces forward.

Reply:
I don't think so. I originally pulled this out for a reason that no longer applies. I'll try putting it back in the advance timeline fn and see if anything breaks.